src/components/PokemonCards.js

```diff
@@ -0,0 +1,43 @@
+import React ,{useState,useEffect} from 'react';
+import axios from "axios";
+
+const PokemonCards = ({name}) => {
+    const [pokemonData,setPokemonData] = useState({});
```

Comment:
Hier zou ik ervoor kiezen om de default waarde `undefined` te maken, zodat je een duidelijker verschil hebt tussen wat wél en niet ingeladen is
```suggestion
    const [pokemonData,setPokemonData] = useState();
```